### PR TITLE
af: Use Application Context for Credentials Manager

### PR DESCRIPTION
**Pull request by poovamraj**
- [x] All new/changed/fixed functionality is covered by tests (or N/A)
- [ ] I have added documentation for all new/changed functionality - N/A - Changes are internal

### 📋 Changes
We are using `applicationContext` to fetch the credentials instead of `Activity` which is not required and could cause issues when credentials are fetched from outside an activity

### 📎 References
https://github.com/auth0/auth0-flutter/issues/233

### 🎯 Testing

We have updated the existing Unit Tests and added check that proper context is set.

### CODE REVIEW

File: auth0_flutter/android/src/main/kotlin/com/auth0/auth0_flutter/CredentialsManagerMethodCallHandler.kt [hunk 1]
```diff
@@ -23,6 +24,8 @@ class CredentialsManagerMethodCallHandler(private val requestHandlers: List<Cred
     // GitHub issue: https://github.com/jacoco/jacoco/issues/1182
     @Generated
     lateinit var activity: Activity
```
Comment (Widcket) [inline auth0_flutter/android/src/main/kotlin/com/auth0/auth0_flutter/CredentialsManagerMethodCallHandler.kt:26]: Do we still need this?

File: auth0_flutter/android/src/test/kotlin/com/auth0/auth0_flutter/CredentialsManagerMethodCallHandlerTest.kt [hunk 2]
```diff
@@ -33,12 +34,14 @@ class CredentialsManagerMethodCallHandlerTest {
         arguments: HashMap<String, Any?> = defaultArguments,
         requestHandlers: List<CredentialsManagerRequestHandler>,
         activity: Activity? = null,
```
Comment (Widcket) [inline auth0_flutter/android/src/test/kotlin/com/auth0/auth0_flutter/CredentialsManagerMethodCallHandlerTest.kt:36]: Is this still needed?

File: auth0_flutter/android/src/main/kotlin/com/auth0/auth0_flutter/CredentialsManagerMethodCallHandler.kt (hunk 1 above)
Comment (poovamraj) [reply auth0_flutter/android/src/main/kotlin/com/auth0/auth0_flutter/CredentialsManagerMethodCallHandler.kt:26]: Yes, as you can see `requireAuthentication` still needs `activity`

File: auth0_flutter/android/src/test/kotlin/com/auth0/auth0_flutter/CredentialsManagerMethodCallHandlerTest.kt (hunk 2 above)
Comment (poovamraj) [reply auth0_flutter/android/src/test/kotlin/com/auth0/auth0_flutter/CredentialsManagerMethodCallHandlerTest.kt:36]: As mentioned in the previous comment, we still use it in `requireAuthentication` so it is required in tests as well

**Review by Widcket — APPROVED** [review]

**Pull request merged**